End2End/functionannotation/src/main/java/com/microsoft/azure/functions/CosmosDB.java

```diff
@@ -0,0 +1,34 @@
+package com.microsoft.azure.functions;
+
+import com.microsoft.azure.functions.annotation.*;
+import java.util.*;
+
+public class CosmosDB {
+    /**
+    * This function will be invoked when a message add to the queue.The documentDB use as input to get the item with the name of the message
```

Comment:
Same goes for all the tests under functionannotation folder. Combine tests with EndToEnd and delete functionannotation folder.